store/src/db/sql.rs

```diff
@@ -82,7 +87,10 @@ pub fn select_accounts_by_block_range(
     let mut stmt = conn.prepare(
         "
         SELECT
-            account_id, account_hash, block_num
+            account_id,
+            account_hash,
+            block_num,
+            details
```

Comment:
This is probably fine for now, but this function is used to build a `sync_state` response where account details are not needed. So, we are doing potentially quite a bit of extra work here by fetching details from the database and parsing them.

Let's create an issue to address this in the future.

Reply:
I refactored this, now there are two different structures (`AccountHashUpdate` is reused), so details aren't provided here anymore.